backend/content/views.py

```
@@ -189,20 +189,17 @@ def perform_create(self, serializer):
     GET  /backend/content/whatsapp-config/  → returns the current WhatsApp link.
-    PATCH /backend/content/whatsapp-config/ → updates the link (board/attivi only).
+    PATCH /backend/content/whatsapp-config/ → updates the link (authorized managers).
     """
     instance = WhatsAppConfig.get_instance()
 
     if request.method == 'GET':
         serializer = WhatsAppConfigSerializer(instance)
         return Response(serializer.data)
 
-    # PATCH – board/attivi or content managers can edit
+    # PATCH – authorized content managers can edit
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)